archetypes/archetype-lambda/src/main/resources/archetype-resources/src/main/java/DependencyFactory.java

```diff
@@ -0,0 +1,27 @@
+#parse ( "global.vm")
+
+package ${package};
+
+import software.amazon.awssdk.auth.credentials.EnvironmentVariableCredentialsProvider;
+import software.amazon.awssdk.http.${httpClientPackageName};
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.${servicePackage}.${serviceClientClassName};
+
+/**
+ * The module containing all dependencies required by the {@link ${handlerClassName}}.
+ */
+public class DependencyFactory {
+
+    private DependencyFactory() {}
+
+    /**
+     * @return an instance of ${serviceClientClassName}
+     */
+    public static ${serviceClientClassName} ${service}Client() {
```

Comment:
In the future, we should consider showing them how to do true DI with dagger.

Reply:
Yup. I thought about adding another parameter, something like `di`, which allows customers to switch between relatively lightweight DI such as `dagger` or `guice` or no DI at all.